Broken on latest nightly

Diesel `0.12` appears to be broken on the latest nightly. I'm seeing the following:

```
error[E0531]: cannot find unit struct/variant or constant `SQLITE_CONSTRAINT_UNIQUE` in module `ffi`
   --> /Users/sbenitez/.cargo/registry/src/github.com-1ecc6299db9ec823/diesel-0.12.0/src/sqlite/connection/stmt.rs:148:14
    |
148 |         ffi::SQLITE_CONSTRAINT_UNIQUE | ffi::SQLITE_CONSTRAINT_PRIMARYKEY =>
    |              ^^^^^^^^^^^^^^^^^^^^^^^^ did you mean `SQLITE_CONSTRAINT`?

error[E0531]: cannot find unit struct/variant or constant `SQLITE_CONSTRAINT_PRIMARYKEY` in module `ffi`
   --> /Users/sbenitez/.cargo/registry/src/github.com-1ecc6299db9ec823/diesel-0.12.0/src/sqlite/connection/stmt.rs:148:46
    |
148 |         ffi::SQLITE_CONSTRAINT_UNIQUE | ffi::SQLITE_CONSTRAINT_PRIMARYKEY =>
    |                                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in `ffi`

error[E0531]: cannot find unit struct/variant or constant `SQLITE_CONSTRAINT_FOREIGNKEY` in module `ffi`
   --> /Users/sbenitez/.cargo/registry/src/github.com-1ecc6299db9ec823/diesel-0.12.0/src/sqlite/connection/stmt.rs:150:14
    |
150 |         ffi::SQLITE_CONSTRAINT_FOREIGNKEY => DatabaseErrorKind::ForeignKeyViolation,
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in `ffi`
```

It doesn't seem like this is something that would be affected by a nightly. Perhaps there are some incorrect version bounds to an FFI library?

--- a/diesel/Cargo.toml
+++ b/diesel/Cargo.toml
@@ -15,7 +15,7 @@ categories = ["database"]
 byteorder = "1.0"
 chrono = { version = "0.3", optional = true }
 clippy = { optional = true, version = "=0.0.121" }
-libsqlite3-sys = { version = ">=0.7.1, <0.8.0", optional = true }
+libsqlite3-sys = { version = ">=0.8.0, <0.9.0", optional = true, features = ["min_sqlite_version_3_7_16"] }
 mysqlclient-sys = { version = ">=0.1.0, <0.3.0", optional = true }
 pq-sys = { version = ">=0.3.0, <0.5.0", optional = true }
 quickcheck = { version = "0.3.1", optional = true }

